components/SpotInfo/style.js

```diff
@@ -117,3 +117,57 @@ export const SpotNameChangedSuccess = styled.div`
   color: #40c057;
   z-index: 100;
 `;
+
+export const ModalHeadline = styled.h2`
+  text-transform: uppercase;
+  font-size: 1.3rem;
+  font-weight: 700;
+  color: #ff0000;
+  margin:0px 0px 10px 0px;
+`;
+
+export const ModalMessage = styled.p`
+  font-size: 1rem;
+  font-weight: 500;
+  color: #000000;
+  margin-top: 0px;
```

Comment:
Minor thing, but if the value if 0, then we do not need the "px". This is generally down to personal preference though.
```suggestion
  margin-top: 0;
```